src/panel/direction/DirectionForm.jsx

```diff
@@ -56,7 +56,17 @@ export default class DirectionForm extends React.Component {
       destinationInputText,
     } = this.props;
 
+    const isMobile = isMobileDevice();
+
+    const VehicleSelectorComponent = props => <VehicleSelector
+      vehicles={vehicles}
+      activeVehicle={activeVehicle}
+      onSelectVehicle={onSelectVehicle}
+      {...props}
+    />;
+
     return <div className="itinerary_form">
+      {!isMobile && <VehicleSelectorComponent />}
```

Comment:
It it's only a display order which changes, you can keep exactly the same DOM and use CSS.
For example, `{display: flex; flex-direction: column-reverse}` on the container will do the trick.

Reply:
yep, done